content/events/hacktoberfest-2020/index-md.md

```diff
@@ -0,0 +1,50 @@
+---
+meta_desc: Join Pulumi at our Infrastructure As Code Workshop at KubeCon EU and
+  learn more about cloud programming, infrastructure as code, and many other
+  topics.
+title: Hacktoberfest 2020
+subtitle: Help us build new open source capabilities for Pulumi
+unlisted: false
+redirect_to: https://hacktoberfest.digitalocean.com
+event:
+  eventbrite_url: " "
+  registration_url: https://hacktoberfest.digitalocean.com
+  calendly_url: " "
+  eventbrite_id: "2"
+  start_date: 2020-10-22
+  time: 9:00 AM - 11:00 AM
+  cost: $0
+  end_date: 2020-10-23
+  location: VIRTUAL
+  type:
+    - Hackathon
+  description: Join the Pulumi team and members of the community as we hack new
+    integrations, automation, and infrastructure libraries!
+type: events
+url_slug: workshop-amsterdam-eu-kubecon-2020
+block_external_search_index: false
+---
+## About Hacktoberfest
+
+Hacktoberfest® is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge.
+
+This year, Pulumi is participating in Hacktoberfest with three areas where the community at large can collaborate to build new open source capabilities:
+
+1. **Native Pulumi Providers** - Want Pulumi to work better with your favorite cloud services? We'll show you how to get started building providers that let Pulumi users take full advantage of these services.
+2. **Operator and Automation API** - Learn how to use Pulumi's new Kubernetes Superpowers to build truly automated deployments for your workloads.
+3. **Reusable Infrastructure Libraries** - Create new abstractions and infrastructure building blocks using Pulumi.
+
+## How to get started
+
+1. **[Register](https://hacktoberfest.digitalocean.com)** on the Hacktoberfest site for access to get credit for your open source contributions.
```

Comment:
@leezen I'll update this link after we publish on Hacktoberfest - it's a chicken and egg problem (both pages need a link referring to each other).